docker/env/hydra.sh

```diff
@@ -118,6 +118,7 @@ if [[ "$1" == "--execute-on-runner" ]]; then
         echo "Going to run a Hydra commands on SCT runner '$SCT_RUNNER_IP'..."
         HOME_DIR="/home/ubuntu"
         echo "Syncing ${SCT_DIR} to the SCT runner instance..."
+        ssh-keygen -R "$SCT_RUNNER_IP" || true
```

Comment:
This patch looks good, can we use a temporary file for each job or runner?

Reply:
probably, but not a trivial solution as this